source/outlines/2016-03-10.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: 20160310
+layout: outline
+---
+
+## All
+
+### Word of the Day
+
+* bowdlerize - transitive verb - to remove or modify the parts (of a book,
+for example) considered offensive.
+
+
+## 1508
+
+### Portfolio Reviews (9:00 - WE FINISH)
+
+In this order:
```

Comment:
Also, I assume Jessica isn't doing a portfolio but it would be good to a. Just wrap up things with her experience and b. pull up her old portfolio for the heck of it.